docs/reference/sql/where.md

```diff
@@ -0,0 +1,48 @@
+`WHERE` clause allows to filter the data that is coming from `FROM` clause of `SELECT`.
+
+# Syntax
+
+```sql
+SELECT *
+FROM table_name
+WHERE condition;
+```
+
+If there is a `WHERE` clause, it must contain an expression with the Boolean type. This is usually
+an expression with comparison and logical operators. Rows where this expression evaluates to false are
+excluded from further transformations or result.
+
```

Comment:
[suggestion]:
If a WHERE clause is present, it should include a Boolean expression, typically involving comparison and logical operators. Any rows for which the expression evaluates to false are excluded from further transformations or the final result